tests/abc_atlas_cache/test_cache.py

```diff
@@ -0,0 +1,573 @@
+        cache.load_manifest(self.old_manifest_string)
+        assert cache._manifest._data == manifest_1
+        assert cache.version == self.old_version
+        assert cache.current_manifest == self.old_manifest_string
+
+        cache.load_manifest(self.new_manifest_string)
+        assert cache._manifest._data == manifest_2
+        assert cache.version == self.new_version
+        assert cache.current_manifest == self.new_manifest_string
+
+        with pytest.raises(ValueError) as context:
+            cache.load_manifest('releases/20200101/manifest.json')
+        msg = 'is not one of the valid manifest names'
+        assert msg in context.value.args[0]
+
+    def test_file_exists(self):
+        """
+        Test that cache._file_exists behaves correctly
+        """
+        data = b'aakderasjklsafetss77123523asf'
+        true_checksum = hash_data(data)
+        test_file_path = self.cache_dir / 'junk.txt'
+        with open(test_file_path, 'wb') as out_file:
+            out_file.write(data)
+
+        cache = S3CloudCache(self.cache_dir, self.test_bucket_name)
+
+        # should be true
+        good_attribute = CacheFileAttributes(url='http://silly.url.com',
+                                             version=self.new_version,
+                                             file_size=1234,
+                                             local_path=test_file_path,
+                                             relative_path='junk.txt',
+                                             file_type='txt',
+                                             file_hash=true_checksum)
+        assert cache._file_exists(good_attribute)
+
+        # test when file path is wrong
+        bad_path = Path('definitely/not/a/file.txt')
+        bad_attribute = CacheFileAttributes(url='http://silly.url.com',
+                                            version=self.new_version,
+                                            file_size=1234,
+                                            local_path=bad_path,
+                                            relative_path='junk.txt',
+                                            file_type='txt',
+                                            file_hash=true_checksum)
+
+        assert not cache._file_exists(bad_attribute)
+
+        # test when path exists but is not a file
+        bad_attribute = CacheFileAttributes(url='http://silly.url.com',
+                                            version=self.new_version,
+                                            file_size=1234,
+                                            local_path=self.cache_dir,
+                                            relative_path='junk.txt',
+                                            file_type='txt',
+                                            file_hash=true_checksum)
+        with pytest.raises(RuntimeError) as context:
+            cache._file_exists(bad_attribute)
+        assert 'but is not a file' in context.value.args[0]
```

Comment:
I suspect this was copy-and-pasted from SDK code written when *somebody* (cough, cough) wasn't totally comfortable with `pytest.raises`. Do you mind fixing this to use `pytest.raises(...., match='but is not a file')`